Cargo.toml

```diff
@@ -28,6 +31,8 @@ serde_json = {version = "1.0.92", features = [ "raw_value" ] }
 clap = { version = "4.1.4", features = [ "derive" ] }
 thiserror = "1.0.38"
 glob = "0.3.1"
+juniper = { version= "0.15.11", default-features = false, features = ["chrono", "schema-language", "url", "uuid"] }
+chrono = { version = "0.4.22", default-features = false, features = ["clock"] }
```

Comment:
Is it possible to use `time` here? We're already using it and the last I recall there was a CVE out on `chrono`

Reply:
Ah yes of course, can do. 👍

Reply:
Actually, although I do agree, I'd propose sticking with chrono for now. Juniper removed the first class integration with `time`, so that'd mean writing custom scalars. Not sure we want to invest effort into that right now.
When it comes to the CVE, we should be good by not including default features.

Reply:
After doing some digging, it looks like `juniper` simply doesn't support `time`. Since this is the case, I agree with Max that it makes more sense to switch to time, thought @mxnnn you would need to replace the usage elsewhere in the code base or create an asana task for this before we could merge this PR

Reply:
`chrono > time`
I'd vote for merging this and removing `time` later. It's not widely used so it will be a small change to do the switcharoo